middle_end/flambda2/tests/mlexamples/digest.ml

```diff
@@ -757,9 +757,7 @@ let file filename =
     d
   | exception e ->
     close_in ic;
-    raise e
-
-let output chan digest = output_string chan digest
+    raise e output_string chan digest
 
```

Comment:
Was this change intended? I don't see how it relates to the commit (and it seems a bit strange; is the point just to be sure `output_string`, `chan`, and `digest` are used?)

Reply:
haha no, that is a mistake left over from `sed` that the compiler didn't catch (!)